courses/utils.py

```diff
@@ -61,3 +75,17 @@ def process_course_run_grade_certificate(course_run_grade):
             user=user, course_run=course_run
         )
         return certificate, created, False
+
+
+def has_equal_properties(course_run_grade, edx_grade):
```

Comment:
This is not necessary, please use `has_equal_properties` from `mitxpro.utils`.

Reply:
oh, I was not aware of already existing method. thanks.